alembic/versions/4b357c65441e_add_ci_status_to_builds.py

```diff
@@ -0,0 +1,38 @@
+"""Add ci_status to builds
+
+Revision ID: 4b357c65441e
+Revises: b01a62d98aa4
+Create Date: 2017-05-11 20:13:41.879435
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '4b357c65441e'
+down_revision = 'b01a62d98aa4'
+
+
+def upgrade():
+    """ Add the ci_status to builds with the corresponding enum. """
+
+    op.execute(
+        "CREATE TYPE ck_ci_status AS ENUM ('ignored', 'queued', 'running', 'passed', 'failed')")
+
+    op.add_column(
+        'builds',
+        sa.Column(
+            'ci_status',
+            sa.Enum('ignored', 'queued', 'running', 'passed', 'failed', name='ck_ci_status'),
+            nullable=True
+        )
+    )
+
+    # Give a default version to all the old builds
+    op.execute("UPDATE builds set ci_status='ignored'")
```

Comment:
If we're giving a default to all builds, we should make the field non-nullable as well. Otherwise a value of NULL is ambiguous.